docs/reference/commandline/service_ps.md

```diff
@@ -49,7 +49,7 @@ bk658fpbex0d   redis.2   redis:3.0.5  worker2   Running        Running 9 seconds
 8eaxrb2fqpbn   redis.10  redis:3.0.5  manager1  Running        Running 8 seconds
 ```
 
-In addition to _running_ tasks, the output also shows the task history. For
+In addition to running tasks, the output also shows the task history. For
```

Comment:
I guess the emphasis on "running" here was to prevent it being confused for "The command allows you to run tasks".

This could be "In addition to tasks that are running" or something along those lines, but we can revisit that in a follow-up.